### PR TITLE
WEBUI-655: add sample workflow with dispatch event

### PR DESCRIPTION
**Pull request by richardsd**
This is a sample workflow that we will use for testing the trigger workflow action. 

I am opening a PR just with this wf, cecause we cannot trigger the `workflow_dispatch` event in workflows that are not in the reference branch.

### CODE REVIEW

File: .github/workflows/dispatch.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: Simple workflow with dispatch trigger
+
+on:
+  workflow_dispatch:
+    inputs:
+      string_parameter:
+        description: 'Simple simple parameter'
```
Comment (Gabez0r) [inline .github/workflows/dispatch.yaml]: nit: I guess a single "simple" would make this "simpler" 😁

**Review by Gabez0r — DISMISSED** [review]

File: .github/workflows/dispatch.yaml (hunk 1 above)
Comment (richardsd) [reply .github/workflows/dispatch.yaml]: 😂 damn, such a small file and with typos

File: .github/workflows/dispatch.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+name: Simple workflow with dispatch trigger
```
Comment (Gabez0r) [inline .github/workflows/dispatch.yaml:3]: Should we add some comments to the top of the file just explaining that is serves the purpose of testing something else?
```suggestion
# ...
# ...
name: Simple workflow with dispatch trigger
```

Comment (richardsd) [reply .github/workflows/dispatch.yaml:3]: Sure, it makes sense, let me add some comments

**Review by semisse — APPROVED** [review]

**Review by mnixo — APPROVED** [review]

**Pull request merged**